### PR TITLE
mgmt, playwrighttesting, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
- https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/playwrighttesting/tests.mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: playwrighttesting
+    Artifacts:
+      - name: azure-resourcemanager-playwrighttesting
+        groupId: com.azure.resourcemanager
+        safeName: azureresourcemanagerplaywrighttesting
+    Clouds: 'Public'
+    UseFederatedAuth: true
```
Comment (weidongxu-microsoft) [inline sdk/playwrighttesting/tests.mgmt.yml:14]: Could you check if this is still used in other ci?

Comment (v-hongli1) [reply sdk/playwrighttesting/tests.mgmt.yml]: fixed by [commit#cfdf4c7](https://github.com/Azure/azure-sdk-for-java/pull/43414/commits/cfdf4c7cc0ceac034556d2d001ba3599b07a253a)

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by XiaofeiCao — APPROVED** [review]

**Comment by XiaofeiCao** [comment]
@v-hongli1 Are you able to see the merge override?
image

**Comment by v-hongli1** [comment]
> @v-hongli1 Are you able to see the merge override? image

No, I can not.
image

**Pull request merged**